### DISCUSSION TITLE
Adjusting Fire Curve Mid-Simulation

### DISCUSSION THREAD

**Discussion by fcurow** [firemodels/fds#12915] [General]
To whom it may concern,

Is it possible under version 6.7.5 to adjust the ramp of a fire mid simulation if the time hasn't reached the time of a ramp line minus 1 line (i.e. the ramp line before to account for the linear interpolation between steps) that is marked for change?

**Comment by drjfloyd** [comment]
You should be able to use a stop file to stop the current simulation and write restart files. Then you can edit the RAMP and RESTART. RAMPs aren't stored in the restart files and as long as all you are doing is taking an existing SURF with a RAMP_Q and changing the RAMP_Q, the restart should work.

FDS 6.7.5 is now four years and eight versions old. I'd recommend you update your version.
